modules/http4s-kernel/test/src/smithy4s/http4s/kernel/Http4sConversionSpec.scala

```diff
@@ -39,7 +39,7 @@ object Http4sConversionSpec extends SimpleIOSuite {
   )
 
   http4sToSmithyAndBackUriTest(
-    uri"example.com",
+    uri"//example.com",
```

Comment:
Without the leading `//`, http4s parses this as the URI path, so the test fails because it was essentially trying to compare `http://localhost/example.com` to `http://example.com`.

Reply:
i've had the "pleasure" of hitting this... yeah it's fin